docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -136,6 +136,26 @@ The great-circle distance between two lat-long points, both specified
 in decimal degrees. An optional final parameter specifies `KM`
 (the default) or `miles`.
 
+### `GREATEST`
+
+Since: 0.18.0
```

Comment:
Should this be 0.20.0? If so, this commit will need to be cherry-picked to the `0.20.x-ksqldb` branch.

Reply:
It should be 0.20.0, I believe we discussed the latter part of this in the sync today.